server/api/users.js

```diff
@@ -39,15 +39,16 @@ router.put('/account', async (req, res, next) => {
         res.sendStatus(200);
       }
     } else {
-      res.redirect('/signup');
+      res.sendStatus(304);
     }
   } catch (error) {
     next(error);
   }
 });
 
 // ADMIN ONLY BELOW
```

Comment:
My understanding of Noelle's suggestion was specifically that we _don't_ create separate backend API routes for admin. Rather, for example, the **/users** route should check if a user is an admin, and if so show all users. If not an admin, it should be blocked. Likewise, **/users/:id** should show the user edit form for user ID = 1 if an admin, or show the user My Account page if the user is logged in and has req.user.id === req.params.id. What do you all think?

Reply:
I agree. Anything a user can do, an admin would also be able to do.
A conditional may need to be used to disallow users from doing anything admins could do.